app/views/miq_ae_tools/_import_export.html.haml

```diff
@@ -157,6 +125,5 @@
     });
 
     ImportSetup.listenForPostMessages(Automate.getAndRenderAutomateJson);
-    ImportSetup.listenForGitPostMessages();
```

Comment:
If this method is no longer needed, can you remove it from the sources?

Reply:
ping @Hyperkid123

Reply:
Ah yes i forgot to remove it. Nice catch!

Reply:
So i am looking at the source code and it turns out this might be needed after all. It is not related to the form i replaced. It seems to render some additional hidden form some jQuery form when uploading from a file. There is no template or controller that would suggest something like this.

I will create new pr ASAP.